app/javascript/crayons/CTAs/__stories__/CTAs.mdx

```diff
@@ -14,10 +13,9 @@ Do not use `<Button />s` as navigational elements. Instead, use Links (or CTAs)
 
 ## Usage
 
-Under the hood, CTA outputs `<a>` element.
+Under the hood, CTA outputs `<a>` element, therefore standard link markup should be used,
+including an `href` (links omitting `href` attributes are not focusable by keyboard).
```

Comment:
nitpick (non-blocking): wording

```suggestion
A CTA renders an anchor element, therefore standard link markup must be used,
including a `href` attribute, otherwise, it is not focusable by the keyboard.
```